final_task/rss_reader/converters.py

```diff
@@ -0,0 +1,170 @@
+import datetime
+import logging
+import os
+
+import requests
+from dominate import document
+from dominate.tags import *
+from fpdf import FPDF
+
+from exceptions_ import ConvertionError
+
+LOGGER = logging.getLogger('rss_logger')
+
+
+def path_validation(path, mode):
+    '''
+    Raises an exception if the path is not valid
+    Otherwise returns path
+
+    Mode is boolean to determine file extention
+    True - .html
+    False - .pdf
+    '''
+    get_extention = (lambda mode: '.html' if mode else '.pdf')
+    LOGGER.debug('CHECKING PATH')
+    if path.endswith('/'):
+        path += 'feed-' + str(datetime.datetime.now()) + get_extention(mode)
+    else:
+        path += '/feed-' + str(datetime.datetime.now()) + get_extention(mode)
+    try:
```

Comment:
For all operations with filesystem paths it is better to use `os.path` module for crossplatform applications.

Reply:
Fixed